### PR TITLE
Add answers to common questions

### PR DESCRIPTION
**Pull request by Ifropc**

### CODE REVIEW

**Review by briwylde08 — APPROVED** [review]

File: docs/anchoring-assets/anchor-platform/sep24/integration.mdx [hunk 1]
```diff
@@ -67,6 +67,31 @@ SECRET_PLATFORM_API_AUTH_SECRET="your encryption key shared with your business s
 
 After it's enabled, all requests must contain a valid `Authorization` header. The JWT provided must have the `jti` and `exp` fields representing a valid transaction and token expiration time, respectively.
 
+## Working With User Request
```
Comment (JakeUrban) [inline docs/anchoring-assets/anchor-platform/sep24/integration.mdx:70]: I'm not sure this is the right place to add this section. The introduction to the "Integration" section only talks about `PATCH /transactions` requests, so adding content on the JWT and `/fee` endpoint seems out of place. Maybe we should move this section to the bottom, or add new FAQ page?

Comment (Ifropc) [reply docs/anchoring-assets/anchor-platform/sep24/integration.mdx:70]: I can move it to a new section, WDYT?

File: docs/anchoring-assets/anchor-platform/sep24/integration.mdx [hunk 2]
```diff
@@ -464,6 +470,24 @@ Finally, after the transaction has been successfully sent and submitted to the n
 
 </CodeExample>
 
+## Extra Transaction Updates
+
+### Expiring Transactions
+
+Your business may want to expire transactions that have been abandoned by the user after some time. It's a good practice to clean up transactions in `inactive` status. To do so, simply change the transaction status to `expired`
```
Comment (JakeUrban) [inline docs/anchoring-assets/anchor-platform/sep24/integration.mdx:477]: We don't have an `inactive` status so I don't think we need that sentence.

File: docs/anchoring-assets/anchor-platform/sep24/faq.mdx [hunk 3]
```diff
@@ -0,0 +1,27 @@
+---
+title: "FAQ"
+sidebar_position: 50
+---
+
+### How To Handle User Request?
```
Comment (JakeUrban) [inline docs/anchoring-assets/anchor-platform/sep24/faq.mdx:6]:
```suggestion
### How To Use Interactive JWTs?
```

**Review by JakeUrban — APPROVED** [review]

File: docs/anchoring-assets/anchor-platform/sep24/integration.mdx (hunk 2 above)
Comment (Ifropc) [reply docs/anchoring-assets/anchor-platform/sep24/integration.mdx:477]: Yeah it has wrong wording, let me fix that

**Pull request merged**